Mathlib/Mem.lean

```diff
@@ -0,0 +1,6 @@
+class Mem (α : outParam $ Type u) (γ : Type v) :=
+(mem : α → γ → Prop)
```

Comment:
```suggestion
class Mem (α : outParam $ Type u) (γ : Type v) where
  mem : α → γ → Prop
```

Reply:
Sure, but why?

Reply:
Because it's the new and fancy syntax, and "all the cool kids use it".

More seriously, I think it's valuable to be consistent with Lean 4 on this.  I don't know why the old syntax is still supported.

Reply:
Oh, I had not realised the new syntax was *replacing* the old one.